js/main.js

```diff
@@ -296,7 +278,7 @@ function calcAdjustedIncome() {
   var result = false;
   try {
     var netEarnedInc = Number($("#netEarnedInc").text());
-    var unearnedInc = Number($("#unearnedInc").text());
+    var unearnedInc = Number($("#unearnedIncTotal").text());
```

Comment:
There is no object named `unearnedInc` and so the resulting `unearnedInc` variable is set to zero, which is a bug.